example/spec/parallel/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -55,7 +55,13 @@ function getReportsKeys(systemBucket, stackName) {
     .then((response) => response.Contents.map((o) => o.Key));
 }
 
+// TODO we will call API to delete the reports when API is updated
```

Comment:
Is there a separate ticket (or PR) to update the API?

Reply:
Yes. CUMULUS-1912